src/python/grpcio_tests/tests/_sanity/_sanity_test.py

```diff
@@ -21,24 +21,25 @@
 import tests
 
 
-class Sanity(unittest.TestCase):
+class SanityTest(unittest.TestCase):
+
+    maxDiff = 32768
```

Comment:
Wait... what is this supposed to do?

Reply:
I mentioned it in the commit log message, though perhaps not in as much detail as I could have for readers who haven't before heard of it. It overrides a `unittest.TestCase` attribute that controls how much to print when an `assert<Something>` method fails. In our case in this test the list of tests can be as much as five whole kilobytes, and I got tired of seeing on test failure the "we've truncated the diff that you're seeing" message. Try it out for yourself by putting a typo into `tests.json` and then running `_sanity_test`?

Reply:
I see. Thanks. Though being able to silently override object *fields* like that (and not as an assignment in *constru*...*em*...initializer) doesn't make me hate python any less.

Reply:
It's uh... it's an object model. Definitely not the one I'd first choose either.